### ISSUE TITLE
Please support "amenity=nightclub"

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#4025]
**[Submitted to the original trac issue database at 3.46pm, Tuesday, 27th September 2011]**

Please support "amenity=nightclub"

The difference between pub and nightclub is, that latter have got a dancefloor. Thus it is inscrutable to me, why pubs are supported in Potlatch but nightclubs are not.

*August 2011: 4700 x amenity=nightclub 
*September 2011: nearly 4800 x amenity=nightclub
*source:  http://taginfo.openstreetmap.org/search?q=nightclub#values

The amount of nightclubs in OSM is progressively growing.

**Comment by tomhughes** [comment]
**Author: Richard**
**[Added to the original trac issue at 3.45pm, Friday, 30th September 2011]**

It's not at all "inscrutable", it's just that no-one has added it. :)

To add this we would need a good-quality icon consistent with the SJJB set that we use (see http://www.sjjb.co.uk/mapicons/contactsheet) in 16x16 and 24x24 sizes (or as an SVG).

**Comment by tomhughes** [comment]
**Author: Tirkon**
**[Added to the original trac issue at 9.13am, Sunday, 9th October 2011]**

Hello Richard,

some suggested icons for nightclubs can be found here:
http://trac.openstreetmap.org/ticket/2697
:-)
Does one of these match your needs?

**Comment by tomhughes** [comment]
**Author: twain**
**[Added to the original trac issue at 3.01pm, Friday, 21st October 2011]**

The icons suggested on #4025 are completely unidentifiable at the sizes we use for the mapnik layer.  We need a better symbol before it is likely to get added.

**Comment by tomhughes** [comment]
**Author: woodpeck**
**[Added to the original trac issue at 3.54pm, Friday, 21st October 2011]**

I beg to differ. We're for example actively using this icon:

http://svn.openstreetmap.org/applications/rendering/mapnik/symbols/department_store.p.16.png

and it comes out ok. Compare this to a 16px version of one of the recommended logos, 

http://www.remote.org/frederik/tmp/ncicon.png

and I think that the latter is ok as well. (I've googled for inspiration about better symbols, but the main themes other than a record player were glasses which we've already used for our bar icon, and dancing people which required a *lot* more detail.)
